src/controllers/__tests__/events.test.js

```diff
@@ -95,4 +95,25 @@ describe("Testing events for routes doesn't require auth controls", () => {
         return events;
       });
   });
+
+  it('GET /api/events/:id should retrieve a single event', (done) => {
```

Comment:
Amazing work! all the tests passed

Reply:
Thanks a lot for your time @yamanrajab90